tsconfig.eslint.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This file was copied over from https://github.com/OctopusDeploy/deploy-release-action/blob/3f894541233fc8c1598aa0f223ce827557003867/tsconfig.eslint.json.